Makefile

```diff
@@ -82,3 +87,16 @@ release: cross
 .PHONY: clean
 clean:
 	rm -rf $(BUILD_DIR)
+
+.PHONY: integration-in-docker
+integration-in-docker:
+	docker build \
+		-f deploy/skaffold/Dockerfile \
+		-t gcr.io/$(GCP_PROJECT)/skaffold-integration .
+	docker run \
+		-v /var/run/docker.sock:/var/run/docker.sock \
+		-v $(HOME)/.config/gcloud:/root/.config/gcloud \
+		-v $(PWD):/go/src/github.com/GoogleCloudPlatform/skaffold \
```

Comment:
nit: reuse the variables for GoogleCloudPlatform/skaffold?

Reply:
fixed